packages/sui-ssr/bin/sui-ssr-archive.js

```diff
@@ -3,14 +3,17 @@
 const program = require('commander')
 const rimraf = require('rimraf')
 const path = require('path')
-
 const archive = require('../archive')
+require('colors')
 
-const OUTPUT_ZIP_PATH = path.join(process.cwd(), 'server.zip')
 const pkg = require(path.join(process.cwd(), 'package.json'))
-
+const REMOVE_ZIP_PATH = path.join(process.cwd(), '*.zip')
```

Comment:
too risky this command!

Reply:
Assuming that the path will be our webapp folder and assuming also that we will have just one zip at time.... will it throw problems?


Don't really know an alternative to that assuming that if we want to add on a next iteration a name param to the command we will have to 'save' the last name in order to remove the 'last' .zip builded.


What will you do in order to change it?

Reply:
If you will use a glob expresión to delete files from the user workspace. At least show and confirm which files you’ll end up deleted.

Reply:
Another thing that we could do is to create all files with a sufix like <name>-build.zip

Reply:
+1 for *-sui-ssr.zip prefix.